coop_memberspace/views/website/statistics.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp><data noupdate="1">
+<odoo>
+    <data noupdate="1">
```

Comment:
```suggestion
```

Please remove all occurrences of `noupdate=1` when they only define QWeb views.
We only want noupdate=1 when we're defining email templates or data